### DISCUSSION TITLE
Please Help, MACOS

### DISCUSSION THREAD

**Discussion by EnVisione** [ORelio/Minecraft-Console-Client#1926] [General]
I managed to get it working on macos but when i join the server i cant send any commands or speak its like the client is just frozen, nothing goes thru. then i just get disconnected.

**Comment by EnVisione** [comment]
envy-MacBook-Air:~ envy$ cd /Users/envy/Desktop/test\ 1
envy-MacBook-Air:test 1 envy$ mono MinecraftClient.exe fakemail@gmail.com fakepassword minesuperior.com "/gserver ms-towny-3"
Minecraft Console Client v1.18.1 - for MC 1.4.6 to 1.18.1 - Github.com/MCCTeam
GitHub build 8, built on 2022-02-05 from commit cc75db7
﻿Resolving minesuperior.com...
Found server play.minesuperior.com:25565 for domain minesuperior.com
Connecting to Minecraft.net...
Retrieving Server Info...
Server is running Forge without mods.
Server version : 1.18 (protocol v1073741883)
Cannot connect to the server : This version is not supported !
Server version :1.18
Restarting Minecraft Console Client...
Cached session is still valid for fake.
Using Minecraft version 1.18 (protocol v757)
Checking if server is running Forge...
Server is running Forge without mods.
Server version : 1.18 (protocol v1073741883)
Server reports a different version than manually set. Login may not work.
Downloading 'en_GB.lang' from Mojang servers...
Done. File saved as 'lang/en_GB.lang'
[MCC] Version is supported.
Logging in...
[MCC] Checking Session...
[MCC] Command  /gserver ms-towny-3  sent.

MINESUPERIOR - LOBBY

Store - store.minesuperior.com
Discord - discord.gg/minesuperior
Vote - minesuperior.com/vote


[MCC] Connection has been lost.
Not connected to any server. Use '/help' for help.
Or press Enter to exit Minecraft Console Client.

**Comment by ReinforceZwei** [comment]
You are running MCC in single command mode, which will connect, send your command, then disconnect. Try to start MCC without the argument `"/gserver ms-towny-3"`.
